docs/user/reference/hook.md

```diff
@@ -221,12 +221,16 @@ In normal operation, a unit will run at least the `install`, `start`, `config-ch
 
 When a charm is first deployed, the following hooks are executed in order before a charm reaches its operation phase:
 
-* storage-attached (if the unit has storage)
+* storage-attached (if the unit has storage) [1]
```

Comment:
Not sure about this referencing technique...

Reply:
> Not sure about this referencing technique...

It's not going to work in the new docs. But let's not get hung up on it -- we'll sort it out together!